<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightDialog</class>
 <widget class="QDialog" name="LightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>251</width>
     <height>430</height>
    </rect>
   </property>
   <property name="title">
    <string>Light</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>230</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Ambient</string>
    </property>
    <widget class="QSlider" name="sliderAR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderAG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderAB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editAR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editAG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editAB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>230</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string>Diffuse</string>
    </property>
    <widget class="QSlider" name="sliderDR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderDG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderDB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>230</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Emissive</string>
    </property>
    <widget class="QSlider" name="sliderER">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderEG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderEB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editER">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>230</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Specular</string>
    </property>
    <widget class="QSlider" name="sliderSR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderSG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderSB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editSR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editSG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editSB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderPow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPow">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>434</y>
     <width>251</width>
     <height>430</height>
    </rect>
   </property>
   <property name="title">
    <string>Material</string>
   </property>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>230</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Ambient</string>
    </property>
    <widget class="QSlider" name="sliderMAR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMAG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMAB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMAR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMAG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMAB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>230</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string>Diffuse</string>
    </property>
    <widget class="QSlider" name="sliderMDR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMDG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMDB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMDR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMDG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMDB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>230</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Emissive</string>
    </property>
    <widget class="QSlider" name="sliderMER">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMEG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMEB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMER">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMEG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMEB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>230</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Specular</string>
    </property>
    <widget class="QSlider" name="sliderMSR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMSG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMSB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMSR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMSG">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMSB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>P :</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMPow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMPow">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
